src/deploy.ts

```diff
@@ -102,7 +102,8 @@ export const deploy = async (
   }
 
   await syncToS3(folder, domainName, cacheBustedPrefix, s3Folder);
-  await invalidateCloudfrontCache(
+
+  await invalidateCloudfrontCacheWithRetry(
```

Comment:
Maybe we should make this behaviour optional ?

Reply:
If it works straight away, it's optional :grin: